test/conformance/values.yaml

```diff
@@ -0,0 +1,97 @@
+# Base set of values common across e2e test scnearios.
+# Please see .github/workflows/conformance.yaml for specific installation
+# input values.
+image:
+  pullPolicy: Always
+
+k8s:
+  requireIPv4PodCIDR: true
+  requireIPv6PodCIDR: false 
+
+
+devices:
+  - eth0
+
+enableIPv4Masquerade: true
+enableIPv6Masquerade: true
+
+debug:
+  enabled: true
+ 
+# Configure unique cluster name & ID (used for ClusterMesh in the future)
+cluster:
+  id: 1
+
+# Routing/encapsulation mode
+tunnelProtocol: vxlan
+tunnel: vxlan
```

Comment:
```suggestion
```
Let's remove this as it's redundant with the `tunnelProtocol` & `routingMode` combo.